terraform/main.tf

```diff
@@ -75,3 +75,45 @@ resource "aws_scheduler_schedule" "data_management_event" {
     role_arn = aws_iam_role.scheduler.arn 
         }
 }
+
+
+# Terraform Resources for the minute-interval Pipeline
+
+# Lambda function that runs the pipeline every minute
+
+resource "aws_lambda_function" "pipeline-lambda" {
+  function_name = "c9-butterflies-pipeline-lambda-tf"
+  role = aws_iam_role.lambda-role.arn
+  image_uri = "129033205317.dkr.ecr.eu-west-2.amazonaws.com/c9-butterflies-pipeline:latest"
+  package_type = "Image"
+  timeout = 540
+  memory_size = 512
+  environment {
+    variables = {
+      AWS_ACCESS_KEY_ID_=var.AWS_ACCESS_KEY_ID_
+      AWS_SECRET_ACCESS_KEY_=var.AWS_SECRET_ACCESS_KEY_
+      BUCKET_NAME=var.BUCKET_NAME
+      DB_HOST=var.DB_HOST
+      DB_NAME=var.DB_NAME
+      DB_PASSWORD=var.DB_PASSWORD
+      DB_PORT=var.DB_PORT
+      DB_SCHEMA=var.DB_SCHEMA
+      DB_USER=var.DB_USER
+    }
+  }
+}
+
+
+# EventBridge Schedule that runs the pipeline lambda every minute
+
+resource "aws_scheduler_schedule" "pipeline-event" {
+  name                = "c9-butterflies-pipeline-schedule-tf"
```

Comment:
I'm being petty, but any reason for this giant gap after name?

Reply:
terraform syntax allows for tabbed alignment of '=' apparently